### PR TITLE
usable example of values

### PR DESCRIPTION
**Pull request by mikeln**
This is the values file that is used to override all other values.

Make it a better example.

### CODE REVIEW

**Comment by mikeln** [comment]
retest this please

File: charts/logging-central/values.yaml [hunk 1]
```diff
@@ -16,12 +16,36 @@ elasticsearch-chart:
...
+      #  lasticsearch-chart.services.es.annotations[0].key="external-dns.alpha.kubernetes.io/hostname",elasticsearch-chart.services.es.annotations[0].value=esold.mikeln.cluster.cnct.io
+      annotations:
+      - key: "external-dns.alpha.kubernetes.io/hostname"
+        value: "es.<newdomain>.cluster.cnct.io" 
+
+  external-dns:
+   # can be set on the commandline as:
+   # domainFilters[0]="mikeln.cluster.cnct.io"
+   # 
+   #  has to match the above withough es
```
Comment (DStorck) [inline charts/logging-central/values.yaml]: withough -> without

File: charts/logging-central/values.yaml [hunk 2]
```diff
@@ -31,6 +55,8 @@ curator:
   name: curator
```
Comment (DStorck) [inline charts/logging-central/values.yaml]: this section needs to be redone to match the current curator filter - changed a few days ago when i added the second filter

File: charts/logging-central/values.yaml [hunk 3]
```diff
@@ -16,12 +16,36 @@ elasticsearch-chart:
...
+  # master_memory_requests: 2Gi
+  # master_Xms_Xmx: 1g
+
+  security:
+    password: "changeme"
+  services:
+    es:
+      public: true
+      # this can be set on the commmand line as; 
+      #  lasticsearch-chart.services.es.annotations[0].key="external-dns.alpha.kubernetes.io/hostname",elasticsearch-chart.services.es.annotations[0].value=esold.mikeln.cluster.cnct.io
```
Comment (DStorck) [inline charts/logging-central/values.yaml]: tiny nit.. missing an 'e' on elasticsearch

**Comment by mikeln** [comment]
NOTE... this values file is not intended to represent ALL the changeable values in the subcharts. It contains the values that are more commonly changed.

**Pull request merged**